stork/scripts/install-format-hook.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+REPO_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/../"
+hookfile="$REPO_DIR/.git/hooks/pre-commit"
+
+if [[ -f $hookfile ]]; then
+    echo "'$hookfile' already exists - aborting" 1>&2
+    exit 1
+fi
+
+echo '#!/bin/bash
+patch=$(git clang-format --diff -- $(git diff --staged --name-only -- src include tests unit))
+if [[ "$patch" =~ "no modified files to format" || "$patch" =~ "clang-format did not modify any files" ]]; then
+    echo "" > /dev/null
+else
+    echo ""
+    echo "Your code has bad style." >&2
```

Comment:
Can we change this error message (and the one in the top level scripts directory) to something a little less confrontational? e.g. "Your code does not follow the MOOSE formatting guidelines."